gravitee-am-reporter/gravitee-am-reporter-kafka/src/main/java/io/gravitee/am/reporter/kafka/dto/AuditMessageValueDto.java

```diff
@@ -37,11 +45,50 @@ public class AuditMessageValueDto {
     private AuditAccessPointDto accessPoint;
     private AuditEntityDto actor;
     private AuditEntityDto target;
-    private String status;
```

Comment:
**question**: Do we really want to remove this field ? IMO this is a Breaking Change as we are changing the current structure of the message. User who are using this reporter will have to adapt the consumer apps. IMO we should keep this field at least in this version. 

I can see two options:
*1/ add the audit details (outcomes message) in this class and remove the AuditOutcomeDto
*2/ keep this status as it is (mark it as deprecated) and add AuditOutcomeDto like you did and use this structure for next evolution

IMO option 2 is the one to go in order to ease the addtion of information in the outcome in the future

wdyt?

Reply:
good call. i'll re-add an `@deprecated` getter so that the field is present in JSON output but not duplicated in code

Reply:
Example log message **after** last change:
```json
{
  "id": "6f06128a-3667-4236-8612-8a36674236f0",
  "transactionId": "e2422ea4-f3d7-4d8b-822e-a4f3d75d8b3e",
  "type": "PASSWORD_POLICY_UPDATED",
  "referenceType": "DOMAIN",
  "referenceId": "d94e531e-2a61-4ba2-8e53-1e2a610ba20c",
  "accessPoint": {
    "alternativeId": "admin",
    "ipAddress": "0.0.0.0",
    "userAgent": "xxxxxxxxxxx"
  },
  "actor": {
    "id": "07ce679a-c1ab-419a-8e67-9ac1ab719abb",
    "alternativeId": "admin",
    "type": "USER",
    "displayName": "Administrator Administrator",
    "referenceType": "ORGANIZATION",
    "referenceId": "DEFAULT"
  },
  "target": {
    "id": "cc8fb38b-38da-4bb2-8fb3-8b38da1bb26b",
    "type": "PASSWORD_POLICY",
    "displayName": "aasd1",
    "referenceType": "DOMAIN",
    "referenceId": "d94e531e-2a61-4ba2-8e53-1e2a610ba20c"
  },
  "outcome": {
    "status": "SUCCESS",
    "message": "[{\"op\":\"replace\",\"path\":\"/name\",\"value\":\"aasd1\"}]"
  },
  "timestamp": "2024-06-28T14:31:16.851124Z",
  "environmentId": "DEFAULT",
  "organizationId": "DEFAULT",
  "domainId": "d94e531e-2a61-4ba2-8e53-1e2a610ba20c",
  "nodeId": "90ae049d-1d39-4e80-ae04-9d1d39ee8002",
  "nodeHostname": "xxxxxxxx",
  "status": "SUCCESS"
}
```